cli/evm.go

```diff
@@ -217,8 +217,8 @@ var EvmDeployCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
-		if argc := cctx.Args().Len(); argc != 1 {
-			return xerrors.Errorf("must pass the contract init code")
+		if cctx.NArg() != 1 {
+			return IncorrectNumArgs(cctx)
 		}
```

Comment:
This is good, but we're losing information here; before this change it tells you what's missing but afterward it just tells you "incorrect number of arguments". So let's improve things: in `cli/helper.go`, just below the `IncorrectNumArgs` function, add a new one:

```go
func IncorrectNumArgsWithHint(cctx *ufcli.Context, hint string) error {
	return ShowHelp(cctx, fmt.Errorf("incorrect number of arguments, got %d, %s", cctx.NArg(), hint))
}
```

Then you can do this:

```suggestion
		if cctx.NArg() != 1 {
			return IncorrectNumArgsWithHint(cctx, "contract init code required")
		}
```